lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -270,6 +284,9 @@ def main():
             retkwargs=retkwargs,
             datastore=datastore,
         )
+        if changed and module.params['save']:
```

Comment:
If `candidate` config and `running` config is same `changed` variable won't be set. In this case `running` config won't be copied to `startup` config even if they are different. I think if `save` option is enabled need to compare between `startup` datastore and `running` datastore and perform `copy_config` only if they are different.